landing-page/components/Footer.tsx

```diff
@@ -1,13 +1,38 @@
 import { useEffect } from 'react'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
 import * as CookieConsent from 'vanilla-cookieconsent'
 import { Span } from '@helpwave/common/components/Span'
 import { Helpwave } from '@helpwave/common/icons/Helpwave'
 import { tw } from '@helpwave/common/twind'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import { useLanguage } from '@helpwave/common/hooks/useLanguage'
+import { Select } from '@helpwave/common/components/user-input/Select'
 import pluginConfig from '../utils/CookieConsentConfig'
 import FooterLinkGroup from './FooterLinkGroup'
-
 import 'vanilla-cookieconsent/dist/cookieconsent.css'
 
+type FooterTranslation = {
+    socials: string,
+    general: string,
+    products: string,
+    development: string
+}
+
+const defaultFooterTranslation: Record<Languages, FooterTranslation> = {
+  en: {
+    socials: 'socials',
+    general: 'general',
+    products: 'products',
+    development: 'development',
+  },
+  de: {
+    socials: 'social',
+    general: 'allgemein',
+    products: 'produkte',
+    development: 'entwicklung',
+  }
```

Comment:
Make the titles uppercase.

Reply:
nit: Why should we differ capitalization between the navigation bar titles and footer titles? The current state of lowercase navbar and uppercase footer seams a little bit inconsistent. I would recommend all lowercase in all languages.

Reply:
In order to create a visual difference of the footer titles (besides bold). Our current state is uppercase and it looks really appealing. We can discuss this on another issues. To me, this issues is not about redesigning - it's about making the current state translated.